content/en/blog/_posts/2022-12-16-statefulset-migration.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: blog
+title: "Kubernetes 1.26: StatefulSet Start Ordinal Simplifies Migration"
+date: 2023-04-19
```

Comment:
```suggestion
date: 2023-04-28
```
Pls also change the file path to match the new date.

Reply:
Thanks! Updated.